WordPress/wordpress-playground
# Remove WP 6.2 minified build after WP 6.6 release

## Motivation for the change, related issues

This PR removes support for WP 6.2 which is unsupported following the WP 6.6 release.

The actual motivation for this change is to try to work around the npm package release error encountered here:
https://github.com/WordPress/wordpress-playground/actions/runs/9960370683/job/27519444281

With this change the `@wp-playground/wordpress-builds` unpacked size goes from 529M to 485M (based on `du` utility output for a local build). Lerna reported that the previous, successful release of the package had a size of 363.5 MB.

Related to #1630

## Testing Instructions (or ideally a Blueprint)

- CI
- Use `npm run dev` to confirm that WP 6.2 is no longer offered as an option in the UI and that Playground continues to load without error.
